.github/workflows/trigger-qa-build.yml

```
@@ -6,16 +6,81 @@ on:
       workflow:
         description: 'Desired Workflow'
         required: true
-        default: 'LernaPublish'
+        default: 'Publish Pre-release'
         type: choice
         options:
-        - LernaPublish
-        - Lerna Publish All Forced
+        - Publish Pre-release
```

```suggestion
        - Publish Pre-release (Recommended)
```

Just so people know to do this first without reading the documentation